pom.xml

```diff
@@ -22,7 +22,7 @@
         <maven.compiler.version>3.11.0</maven.compiler.version>
         <maven.deploy.version>2.8.2</maven.deploy.version>
         <maven.release.version>2.5.3</maven.release.version>
-        <okapi.version>1.45.0</okapi.version>
+        <okapi.version>1.46.0</okapi.version>
```

Comment:
We need to test that output from push & pull commands is identical across different versions after updating okapi

Reply:
I have tested it against mojito dev, and it is working as expected.